Documentation/data-collection.md

```diff
@@ -475,9 +475,47 @@ data:
     #
     # owners: (@openshift/networking)
     #
-    # ovnkube_master_egress_routing_via_host" informs if the OVN-K cluster's gateway mode is
+    # cluster:ovnkube_controller_egress_routing_via_host:max informs if the OVN-K cluster's gateway mode is
     # `routingViaOVN` (0), `routingViaHost` (1) or invalid (2).
-    - '{__name__="cluster:ovnkube_master_egress_routing_via_host:max"}'
+    - '{__name__="cluster:cluster:ovnkube_controller_egress_routing_via_host:max"}'
```

Comment:
```suggestion
    - '{__name__="cluster:ovnkube_controller_egress_routing_via_host:max"}'
```

Reply:
oh nice catch @juzhao ! thank you, let me see if this was wrongly done here: https://github.com/openshift/cluster-monitoring-operator/pull/2297/files#diff-673e680d1eeb9389f27aced5cccc9148a722c103c0bae4d49641c61559f5c76cR480
I think its looking good on master, so its a problem only here I guess. phew